docs/getting-started/install-elastic-endpoint-ven.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[deploy-elastic-endpoint-ven]]
+= Install {elastic-endpoint} manually on macOS Ventura and higher
+
+To properly install and configure {elastic-endpoint} manually without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the endpoint before {elastic-endpoint} can be fully functional:
+
+* <<system-extension-endpoint, Approve the system extension>>
+* <<allow-filter-content, Approve network content filtering>>
+* <<enable-fda-endpoint, Enable Full Disk Access>>
+
+NOTE: The following permissions that need to be enabled are required after you <<install-endpoint, configure and install the {elastic-defend} integration>>, which includes <<enroll-security-agent, enrolling the {agent}>>.
+
+[discrete]
+[[system-extension-endpoint-ven]]
+== Approve the system extension for {elastic-endpoint}
+
+For macOS Ventura (13.0) and later, {elastic-endpoint} will attempt to load a system extension during installation. This system extension must be loaded in order to provide insight into system events such as process events, file system events, and network events.
+
+The following message appears during installation:
+
+[role="screenshot"]
+image::install-endpoint-ven/system_extension_blocked_warning_ven.png[]
+
+. Click *Open Security Settings*.
+. In the left pane, click *Privacy & Security*.
++
+[role="screenshot"]
+image::install-endpoint-ven/privacy_security_ven.png[]
++
+// We'll need to make ElasticEndpoint a new shared attribute and then add that attribute throughout this file.
```

Comment:
I'm not sure we need a shared attribute here, because the text isn't being used anywhere else besides this one file.

Reply:
```suggestion
```